index.js

```diff
@@ -106,6 +102,8 @@ module.exports = fp(async function (app, opts) {
   fastifyGraphQl.extendSchema = function (s) {
     if (typeof s === 'string') {
       s = parse(s)
+    } else {
```

Comment:
```suggestion
    } else if (!s || typeof s !== 'object') {
```

This will also need an updated test. Essentially if the user has already passed in a parsed AST, we should support it.